### ISSUE TITLE
Update the arxiv submission once it is done

### ISSUE THREAD

**Issue by francisbrochu** [francisbrochu/francisbrochu.github.io#3]

**Comment by francisbrochu** [comment]
Currently on hold. Also possible to upload the pdf on the site directly, by creating a pdfs folder and linking there.

**Comment by francisbrochu** [comment]
arxiv posted and entries added.
